events/code_review_events/workflow.py

```diff
@@ -314,25 +357,31 @@ def __init__(self, cache_root):
             self.webserver = WebServer(QUEUE_WEB_BUILDS)
             self.webserver.register(self.bus)
 
-            # Create pulse listener for unit test failures
+            # Create pulse listener
+            exchanges = []
+            if taskcluster_config.secrets["autoland_enabled"]:
+                logger.info("Autoland ingestion is enabled")
+                exchanges += [
+                    # autoland ingestion
+                    (PULSE_TASK_GROUP_RESOLVED, "#")
```

Comment:
You can restrict to level 3 task groups, by doing `#.gecko-level-3.#`, so at least you'll only have autoland and mozilla-central

Reply:
Awesome